docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -99,7 +99,7 @@ their reliability while chasing for faster pipelines. There are requirements to
 with trace visualizations to show what’s happening and improve performance and reliability
 (flaky tests and pipelines).
 
-Integrating with many popular CIDevOps tools like Maven or Ansible using OpenTelemetry, Elastic Observability
+Integrating with many popular CICD/DevOps tools like Maven or Ansible using OpenTelemetry, Elastic Observability
```

Comment:
This is the only place in the doc where we say CICD rather than CI/CD. Moreover, in the first sentence of the doc, we defined CI/CD="Continuous Integration and Continuous Delivery"